test/jdk/java/foreign/TestClassLoaderFindNative.java

```diff
@@ -59,7 +65,13 @@ public void testInvalidSymbolLookup() {
     @Test
     public void testVariableSymbolLookup() {
         MemorySegment segment = SymbolLookup.loaderLookup().find("c").get().reinterpret(1);
-        assertEquals(segment.get(JAVA_BYTE, 0), 42);
+        /* JAVA_INT applies to both Little-Endian and Big-Endian
+         * platforms given the one-byte int value is stored at the
+         * highest address(offset 3) of the int type in native on
+         * the Big-Endian platforms.
+         * See libLookupTest.c
+         */
```

Comment:
The comment is misleading. The variable being accessed is of type `int` (not likely to be one byte). Suggest
```java
        /* The variable is 'int c;', so JAVA_INT is a better choice than JAVA_BYTE.
         * See libLookupTest.c.
         */
```

Reply:
Updated